sdk/communication/communication-chat/src/constants.ts

```diff
@@ -1,4 +1,4 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-export const SDK_VERSION: string = "1.0.0-beta.5";
+export const SDK_VERSION: string = "1.0.0-beta.6";
```

Comment:
This change is already in master, but it won't cause a merge conflict because it's identical.